source/codegen/CodeGenerator.cpp

```diff
@@ -86,8 +86,13 @@ GeneratedCode CodeGenerator::finish() {
 
     // Verify all generated code.
     bool bad = llvm::verifyModule(*module, &llvm::errs());
-    if (bad)
+    if (bad) {
+#ifdef __APPLE__
```

Comment:
This can probably just be unified and have all platforms call print()

Reply:
Yes, I remove #ifdef __APPLE__ and apply the replacement for all the platforms.